psi4/src/psi4/detci/ciwave.cc

```diff
@@ -747,6 +747,8 @@ void CIWavefunction::semicanonical_orbs() {
 
     // Allocate unitary transformation (only for DOCC + ACTV + VIR orbs)
     auto U = std::make_shared<Matrix>("U to semi", nrotpi, nrotpi);
+    // zero U otherwise we get small errors
+    U->zero();
```

Comment:
Can we extend one of the tests to ensure the the root value and final value match exactly?

Reply:
Ok, I added the case that fails and checked for the energy.